builder/qemu/step_type_boot_command.go

```diff
@@ -157,11 +167,7 @@ func vncSendString(c *vnc.ClientConn, original string) {
 			log.Printf("Special code '<leftAltOn>' found, replacing with: %d", keyCode)
 
 			c.KeyEvent(keyCode, true)
-			time.Sleep(time.Second / 10)
```

Comment:
Looks like the duped code was added (in its current form) in a87ad05866346d6c610fa49caf5bdb72e6877975

Since the VNC implementation is shared with VMware and QEMU, and we have been using 100ms with VMware, I think this is probably OK.

In the worst case a QEMU user can set this to 200ms using the ENV var.